code/game/machinery/suit_storage_unit.dm

```diff
@@ -165,7 +165,48 @@
 		playsound(loc, I.usesound, 100, 1)
 		to_chat(user, text("<span class='notice'>You [panel_open ? "open up" : "close"] the unit's maintenance panel.</span>"))
 		updateUsrDialog()
+		return
+		
+	if(!state_open)
+		to_chat(user, "<span class ='warning'>It's closed!</span>")
+		return
 
+	if(istype(I, /obj/item/clothing/head/helmet)) // Helmets
+		if(helmet)
+			to_chat(user, "<span class='warning'>The unit already contains a helmet!</span>")
+			return
+		user.drop_item()
+		I.forceMove(src)
+		helmet = I
+		update_icon()
+		updateUsrDialog()
+		to_chat(user, "<span class='notice'>You load the [I] into the storage compartment.</span>")

```

Comment:
Putting `the` before `[src]` will result in `the the suit storage unit` to display

Reply:
it's for the space suits and it doesnt double the
https://i.imgur.com/NRuwDOx.png

Reply:
Ahh okay 👍